database/src/main/java/com/firebase/ui/database/FirebaseArray.java

```diff
@@ -20,25 +20,26 @@
 import com.google.firebase.database.Query;
 
 import java.util.ArrayList;
-import java.util.List;
+import java.util.HashMap;
 
 /**
  * This class implements an array-like collection on top of a Firebase location.
  */
 class FirebaseArray implements ChildEventListener {
-    public interface OnChangedListener {
+    interface OnChangedListener {
```

Comment:
@CodingDoug why change the visibility on this `interface` as well as the constructor below? I believe @puf just got done approving this particular visibility.